### PR TITLE
Refactor/20240409/moves qub docs to ulisboa

### PR DESCRIPTION
**Pull request by joaofreitas-qubit**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -138,13 +138,15 @@
...
-			<artifactId>fenixedu-commons</artifactId>
+			<groupId>org.freemarker</groupId>
+			<artifactId>freemarker</artifactId>
+			<version>2.3.20</version>
 		</dependency>
 		<dependency>
-			<groupId>com.qubit.terra</groupId>
-			<artifactId>qub-docs</artifactId>
+			<groupId>org.fenixedu</groupId>
+			<artifactId>fenixedu-commons</artifactId>
```
Comment (pabrantes) [inline pom.xml:149]: @joaofreitas-qubit why are we adding fenixedu-commons?

**Review by pabrantes — CHANGES_REQUESTED** [review]
I get why the qub-docs dependency is being removed, but not really sure why it's being replaced by fenixedu-commons artifact. Can you elaborate on that?

File: pom.xml (hunk 1 above)
Comment (joaofreitas-qubit) [reply pom.xml:149]: Actually the dependency was already there, it seems that it just switched places with the freemarker one

**Review by anilmamede — APPROVED** [review]
I reviewed the changes and it is fine

**Pull request merged**